internal/driver/rowbuilder.go

```
@@ -0,0 +1,139 @@
+			body.WriteWrappedCall(indent+1, embedHead, embedArgs, "),")
+
+			continue
+		}
+		body.WriteIndentedLine(indent+1, rb.formatColumnValue(col, idx)+",")
+		idx++
+	}
+	body.WriteIndentedLine(indent, ")")
+}
+
+// formatEmbedConstruction returns "name=EmbedType(field1=row[i], ...)".
+func (rb *RowBuilder) formatEmbedConstruction(col model.Column, idx *int) string {
```

golangci-lint (2.12.2)

[error] unexported method "formatEmbedConstruction" for struct "RowBuilder" should be placed after the exported method "WriteScalarReturn"

(funcorder)